app/models/article.rb

```diff
@@ -1,29 +1,38 @@
 # frozen_string_literal: true
 
 class Article < ApplicationRecord
+  TITLE_LENGTH_MAX = 50
   VALID_STATUSES = {
     draft: 0,
     published: 1,
     trashed: 2
   }.freeze
 
   LANGUAGES = {
-    ukr: 0,
-    eng: 1
+    ua: 0,
+    en: 1
   }.freeze
 
-  enum status:   VALID_STATUSES, _prefix: true
-  enum language: LANGUAGES, _prefix: true
+  enum status:   VALID_STATUSES
+  enum language: LANGUAGES
 
-  has_many    :article_tags
+  belongs_to  :user, foreign_key: 'user_id', inverse_of: :articles
+  has_many    :article_tags, dependent: :destroy
   has_many    :tags, through: :article_tags
-  belongs_to  :user
 
-  default_scope -> { order(created_at: :desc) }
+  validates   :title, :text, :user, :status, :language, presence: true
+  validates   :title, length: { maximum: TITLE_LENGTH_MAX }
 
-  validates   :title, presence: true, length: { maximum: 50 }
-  validates   :text, presence: true
-  validates   :user, presence: true
-  validates   :status, presence: true
-  validates   :language, presence: true
+  # This is required for the following function in another PR
+  # scope :sorted_desc, -> { order(created_at: :desc) }
+
+  def tag_list=(tags_string)
```

Comment:
Oh no, we don't do that, keep your model clean. Create Service for that, if you need this logic. (app/services)

Reply:
@mmrshk this method moved in form